spec/unit/blockscore/person_spec.rb

```diff
@@ -1,10 +1,12 @@
 module BlockScore
   RSpec.describe Person do
     describe '.new' do
-      subject(:person) { BlockScore::Person.new(attributes_for(:person)) }
+      let(:attrs)      { attributes_for(:person) }
+      subject(:person) { BlockScore::Person.new(attrs) }
```

Comment:
The subject is evaluated lazily, so there's no particular need to define the `:attrs` let block before the subject.

Typically I will order my spec setup like:

* subject
* let
* before
* after
* it/its blocks

The subject is the object under test, so I think it's the most important thing in the test and what should appear first.